src/content/en/_index.yaml

```diff
@@ -60,7 +60,7 @@ landing_page:
         </p>
         <ul class="checkmarks">
           <li>
-            <b>Reliable</b> - Load instantly and never show the downasaur,
+            <b>Reliable</b> - Load instantly and never show the dinosaur,
```

Comment:
So, I've been on Chromium for 3 years and never heard it called this. [A Google search](https://www.google.com/search?q=chrome+%22downasaur%22) turns up few results, and the only results for site:google.com are from this copy. It's not in the Chromium code base either. Is this something web developers would be familiar with?

I'll leave this change out of my next push and let y'all decide whether to change this later.